homeassistant/components/media_player/universal.py

```diff
@@ -328,6 +332,11 @@ def current_source(self):
         return self._child_attr(ATTR_INPUT_SOURCE)
 
     @property
+    def source_list(self):
+        """List of available input sources."""
+        return self._source_list
```

Comment:
Wouldn't it make more sense to return the sources list of the active media player ?